doc/matchers.md

```diff
@@ -292,4 +292,13 @@ For the extension function style, each function has an equivalent negated versio
 | `kcallable.shouldAcceptParameters(parameters) { block }` | Asserts that the member can be called with the parameters (check the types), and then, runs the block with the annotation |
 | `kcallable.shouldHaveParametersWithName(parameters)` | Asserts that the member has the parameters with the given name |
 | `kcallable.shouldHaveParametersWithName(parameters) { block }` | Asserts that the member has the parameters with the given name, and then, runs the block with the annotation |
-| `ktype.shouldBeOfType<T>()` | Asserts that the KType has the type T | 
+| `ktype.shouldBeOfType<T>()` | Asserts that the KType has the type T |
+
+| Instant ||
+| -------- | ---- |
+| `instant.shouldBeAfter(anotherInstant)` | Asserts that the instant is after anotherInstant |
+| `instant.shouldNotBeAfter(anotherInstant)` | Asserts that the instant is not after anotherInstant which means the instants are either equal or instant is before anotherInstant |
+| `instant.shouldBeBefore(anotherInstant)` | Asserts that the instant is before anotherInstant |
+| `instant.shouldNotBeBefore(anotherInstant)` | Asserts that the instant is not before anotherInstant which means the instants are either equal or instant is after anotherInstant |
+| `instant.shouldBeBetween(fromInstant, toInstant)` | Asserts that the instant is between fromInstant and toInstant |
+| `instant.shouldNotBeBefore(fromInstant, toInstant)` | Asserts that the instant is not between fromInstant and toInstant |
```

Comment:
We don't document `shouldNot`, as it's always expected to have a `not` matchers. Could you remove them, please?

Reply:
sure